entries/b/buyee.jp.json

```
@@ -0,0 +1,13 @@
+{
+  "Buyee": {
+    "domain": "buyee.jp",
+    "contact": {
+      "facebook": "buyee.jp.en",
+      "twitter": "Buyee",
+      "email": "support@buyee.jp"
+    },
+    "keywords": [
+      "other"
```

Even though they don't sell the goods themselves, the `retail` category is a better fit, I guess.
```suggestion
      "retail"
```